### PR TITLE
Add May 2023 monthly update.

### PR DESCRIPTION
**Pull request by kgpai**

### CODE REVIEW

**Comment by kgpai** [comment]
Gist: https://gist.github.com/kgpai/db9bcf80059b9bd24d05adb0bd3ecf62

File: velox/docs/monthly-updates/may-2023.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
+********************
+May 2023 Update
+********************
```
Comment (majetideepak) [inline velox/docs/monthly-updates/may-2023.rst:3]: nit: end the lines where the text ends.
```
***************
May 2023 Update
***************
```

File: velox/docs/monthly-updates/may-2023.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Documentation
+=============
+
+ * Add `blog post on efficient complex types <https://velox-lib.io/blog/simple-functions-2>`_.
+
+
+Core Library
+============
+
+ * Add support to implement DECIMAL types as logical type.
```
Comment (majetideepak) [inline velox/docs/monthly-updates/may-2023.rst:14]: `DECIMAL types` -> `DECIMAL type`
I know the PR title says `types`. That should have been `type` as well.

File: velox/docs/monthly-updates/may-2023.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+ * Add support for retrieve signatures of potential companion functions of a UDAF.
```
Comment (majetideepak) [inline velox/docs/monthly-updates/may-2023.rst:15]: `Add support to retrieve` or `Add support for retrieving`

File: velox/docs/monthly-updates/may-2023.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+ * Add window function :func:`kNullRow` .
+ * Optimize :func:`map` for constant keys.
+
+
+
+Spark Functions
+===============
+
+ * Add :spark:func:`asinh`, :spark:func:`acosh`, :spark:func:`atanh`, :spark:func:`sec`, :spark:func:`csc` functions.
+ * Add :spark:func:`make_date` function.
```
Comment (majetideepak) [inline velox/docs/monthly-updates/may-2023.rst:39]: add `make_date` to the previous item as well and remove this?

**Review by majetideepak — APPROVED** [review]
Thanks, @kgpai

**Review by bikramSingh91 — APPROVED** [review]

**Pull request closed without merging**